lib/msal-core/src/Storage.ts

```diff
@@ -139,20 +139,20 @@ export class Storage {// Singleton
     }
 
     setItemCookie(cName: string, cValue: string, expires?: number): void {
-        var cookieStr = cName + "=" + cValue + ";";
+        let cookieStr = cName + "=" + cValue + ";";
         if (expires) {
-            var expireTime = this.setExpirationCookie(expires);
+            const expireTime = this.setExpirationCookie(expires);
             cookieStr += "expires=" + expireTime + ";";
         }
 
         document.cookie = cookieStr;
     }
 
     getItemCookie(cName: string): string {
-        var name = cName + "=";
-        var ca = document.cookie.split(";");
-        for (var i = 0; i < ca.length; i++) {
-            var c = ca[i];
+        const name = cName + "=";
+        const ca = document.cookie.split(";");
+        for (let i = 0; i < ca.length; i++) {
+            let c = ca[i];
             while (c.charAt(0) === " ") {
```

Comment:
why do we need to have this while loop at all? either the string contains the name or not

Reply:
its possible i don't understand this code / intent, its just a little confusing

Reply:
Storage code needs some cleanup as we have a bug fix which is pipelined #517 . May be we can take a look at the cookie storage when we handle the same. I can make a note.